File: ProgramingBasics/SimpleCalculations/10.RadiansToDegrees/src/RadiansToDegrees.java
import java.util.Scanner;

public class RadiansToDegrees {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double radians = Double.parseDouble(scanner.nextLine());
        double celsius = radians * 180 / Math.PI;

        System.out.println(Math.round(celsius));
    }
}

File: ProgramingBasics/SimpleCalculations/10.RadiansToDegrees/10.RadiansToDegrees/RadiansToDegrees.cs
using System;

namespace _10.RadiansToDegrees
{
    public class RadiansToDegrees
    {
        public static void Main(string[] args)
        {
            var radians = double.Parse(Console.ReadLine());
            var celsius = radians * 180 / Math.PI;

            Console.WriteLine(Math.Round(celsius));
        }
    }
}
